components/Main/Workouts/Workouts.js

```diff
@@ -1,29 +1,67 @@
 import { StyledWorkouts, StyledWorkoutsHeader } from "./StyledWorkouts";
 import Image from "next/image";
-import Link from "next/link";
+import InputWorkout from "./InputWorkout/InputWorkout";
+import { useState } from "react";
+import React from "react";
+
+const Workouts = ({ workouts, onAddWorkout, onDelete }) => {
+  const [showInput, setShowInput] = useState(false);
+
+  function toggleShow() {
+    setShowInput(true);
+  }
+
+  async function handleSubmit(event) {
+    event.preventDefault();
+    event.stopPropagation();
+    const name = event.target.elements.name.value;
+    const newWorkout = {
+      name: name,
+    };
+
+    await fetch("/api/workouts", {
+      method: "POST",
+      headers: {
+        "Content-Type": "application/json",
+      },
+      body: JSON.stringify(newWorkout),
+    });
+    onAddWorkout(newWorkout);
+    event.target.reset();
+    setShowInput(false);
+  }
 
-const Workouts = ({ workouts }) => {
   return (
     <>
       <StyledWorkoutsHeader>
         <h2>My-Workouts</h2>
-        <Link href="/workouts/addWorkouts">
+        <button onClick={() => toggleShow()}>
           <Image
             className="add-workout"
             src="/plus-circle.svg"
             width={32}
             height={32}
-            alt="add-workout"
+            alt="show input"
           />
-        </Link>
+        </button>
       </StyledWorkoutsHeader>
+      {showInput ? <InputWorkout onSubmit={handleSubmit} /> : null}
```

Comment:
das ist eine sehr coole Animation :) ich würde aber die Ecken des Headers dann nicht mehr abrunden... 
<img width="342" alt="Bildschirmfoto 2023-01-17 um 18 06 31" src="https://user-images.githubusercontent.com/115288914/212964956-68d1e1d8-cf00-4d81-b02c-05df166b2f37.png">

Reply:
Ich versuche die border bottom mit aktivem input zu deaktivieren und mit inaktivem input wieder abzurunden. Bin überfragt wie, widme mich dem zu einem späteren Zeitpunkt :)

Reply:
du hast ja showInput schon im State, du könntest ihn den Komponenten mitgeben und dann den border-radius in eine Klasse "border" packen und abhängig vom State die Klasse zufügen oder nicht.... so wie [hier](https://github.com/ufelgen/capstone-project/blob/main/components/ConjugationButtons/ConjugationButtons.js), das wäre dann `className={showInput ? "" : "border"}` oder so...